contracts/interfaces/ICover.sol

```diff
@@ -98,7 +98,24 @@ interface ICover {
 
   /* ========== VIEWS ========== */
 
-  function covers(uint id) external view returns (uint24, uint8, uint96, uint32, uint32, uint16);
+  //function coverData(uint id) external view returns (
+    //uint24 productId,
+    //uint8 payoutAsset,
+    //uint96 amountPaidOut
+  //);
+
+  function coverData(uint id) external view returns (CoverData memory);
+
+  function coverSegmentsCount(uint coverId) external view returns (uint);
+
+  //function coverSegments(uint coverId, uint segmentId) external view returns (
```

Comment:
is this left commented out to be enabled later?

Reply:
if not let's delete

Reply:
deleted